views/listsitesforlocale.inc.php

```diff
@@ -172,6 +172,14 @@
                                "     Example: </p><blockquote>;Plugins<br/>Plugins {ok}</blockquote>\n" .
                                "  </div>\n";
             }
+
+            if ($locale_analysis['untranslated_var'] || LangManager::countErrors($locale_analysis['errors'], 'python')) {
+                $todo_files .= "  <div class='tip'>\n" .
+                               "    <p><strong>Tip:</strong> This file contains variables that can use following formats:  \n" .
+                               "    </p><blockquote><strong>%(variable)s</strong>, <strong>%s</strong>, or <strong>%%</strong> (“escaped” percentage sign).</blockquote>\n" .
+                               "    <p>Make sure that they remain as they are in English.</p>\n" .
```

Comment:
Alternative text: `Make sure to keep these variables unchanged: don’t translate the text between brackets, or remove the “s” after the closing bracket.`